tests/gpaw/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# add tests for GPAW4QMCPACK converter
+function(ADD_GPAW4QMCPACK_TEST test_name)
+      set(EXE_NAME ${qmcpack_BINARY_DIR}/bin/gpaw4qmcpack)
+  execute_process(COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_CURRENT_SOURCE_DIR}/gpaw4qmcpack_test.py"
```

Comment:
Why does this need to happen at every ADD_GPAW4QMCPACK_TEST?

Reply:
You're right, it probably doesn't. It's an artifact from a simpler test case for the test. I'll move it outside of the function.